docs/openapi/lpa-codes-openapi-v1.yml

```diff
@@ -290,34 +306,35 @@ components:
   schemas:
     CodeCreated:
       properties:
-        code:
-          type: string
-        status:
-          type: string
-        id:
-          type: string
-      required:
-        - code
-    CodeRevoked:
-      properties:
-        code:
-          type: string
-        status:
-          type: string
-        id:
-          type: string
+        codes:
+          type: array
+          items:
+            type: object
+            properties:
+              lpa:
+                type: string
+              actor:
+                type: string
+              code:
+                type: string
+            example:
+                codes:
+                - lpa: "eed4f597-fd87-4536-99d0-895778824861"
+                  actor: "12ad81a9-f89d-4804-99f5-7c0c8669ac9b"
+                  code: "YsSu4iAztUXm"
+                - lpa: "eed4f597-fd87-4536-99d0-895778824861"
+                  actor: "9a619d46-8712-4bfb-a49f-c14914ff319d"
+                  code: "aEYVS6i9zSwy"
```

Comment:
I think the indentation here isn't quite right, it's showing funny in Swagger Editor:
<img width="585" alt="image" src="https://user-images.githubusercontent.com/100852/81177439-8fe55e80-8f9e-11ea-82e8-aca7cc5a08a2.png">

I suspect `example` should probably be aligned with `items`

Reply:
Fixed.